ccd-claim-migration/src/main/java/uk/gov/hmcts/cmc/ccd/migration/services/ClaimMigrator.java

```diff
@@ -100,14 +109,26 @@ private void migrateClaims(
         AtomicInteger failedOnUpdateMigrations
     ) {
         notMigratedClaims.parallelStream().forEach(claim -> {
-            migrationHandler.migrateClaim(
-                migratedClaims,
-                failedOnCreateMigrations,
-                failedOnUpdateMigrations,
-                updatedClaims,
-                claim,
-                user
-            );
+            if (fixDataIssues && isSettledOrJudgement(claim)) {
+                dataFixHandler.fixClaim(migratedClaims, failedOnUpdateMigrations, updatedClaims, claim, user);
+            } else {
+                migrationHandler.migrateClaim(
```

Comment:
Just for this run, can we comment this migrateClaim... ? If by mistake we send a claim which is not in if condition, this will duplicate the claim by migrating again.